app/translations.js

```diff
@@ -220,43 +220,49 @@ const translations = {
   'sv': {
     'frontpage': 'Framsidan',
     'stop-page.title-short': 'Hållplats',
+    'stop-page.title': 'Hållplats {stop_name} - {stop_code}',
```

Comment:
We probably should start the translation fixes by enabling eslint rule `sort-keys` for this file and committing the messages in correct order, in order to make tracking changes like this easier.

Also, we probably should group these messages somehow.